src/main/java/DummyCommand.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Simulates a dummy command of the Duke System.
+ * This is to be used to signify an error.
```

Comment:
Understand where you are coming from with the use of Dummy commands, but perhaps it is more appropriate to handle the error directly using exceptions!